docs/using/recipes.md

```diff
@@ -485,13 +487,15 @@ In this case, you should use the `start.sh` script to launch the server with no
 For jupyterlab:
 
 ```bash
-docker run jupyter/base-notebook:b418b67c225b start.sh jupyter lab --LabApp.token=''
+docker run jupyter/base-notebook:b418b67c225b \
+        start.sh jupyter lab --LabApp.token=''
 ```
 
 For jupyter classic:
 
 ```bash
-docker run jupyter/base-notebook:b418b67c225b start.sh jupyter notebook --NotebookApp.token=''
+docker run jupyter/base-notebook:b418b67c225b \
+        start.sh jupyter notebook --NotebookApp.token=''
```

Comment:
Also 4 spaces :)

Reply:
I was not super sure.  I saw two different indentations across files so just chose one. Will fix them all